app/code/Magento/Ui/view/base/web/js/grid/url-filter-applier.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'uiComponent',
+    'underscore'
+], function (Component, _) {
+    'use strict';
+
+    return Component.extend({
+        defaults: {
+            filterProvider: 'componentType = filters',
```

Comment:
how this component is behaving on a page with multiple grids?
I.e. media gallery + adobe stock

Reply:
Good point, will have a look on this, probably will have to introduce a parameter to handle the component namespace.